es6-shim.js

```diff
@@ -1536,11 +1406,22 @@
     },
 
     fround: function fround(x) {
-      if (x === 0 || x === Infinity || x === -Infinity || Number.isNaN(x)) {
+      if (x === 0 || x === Infinity || x === -Infinity || numberIsNaN(x)) {
         return x;
       }
-      var num = Number(x);
-      return numberConversion.toFloat32(num);
+      var v = Number(x);
```

Comment:
https://people.mozilla.org/~jorendorff/es6-draft.html#sec-function-properties-of-the-math-object https://people.mozilla.org/~jorendorff/es6-draft.html#sec-math.acosh - good call, I'll move that line higher.